src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/TreeNodeCollectionTests.cs

```diff
@@ -80,6 +80,52 @@ public void TreeNodeCollection_Item_SetInvalidIndex_ThrowsArgumentOutOfRangeExce
             Assert.Throws<ArgumentOutOfRangeException>("index", () => collection[index] = new TreeNode());
         }
 
+        [WinFormsFact]
+        public void TreeNodeCollection_Item_SetTreeNodeAlreadyAdded_ThrowsArgumentException()
```

Comment:
This test doesn't throw exceptions, hence the name is incorrect.
```suggestion
        public void TreeNodeCollection_Item_SetSameNodeAgain_Noop()
```

Reply:
OMG, just fixed it, thank you